site/content/en/docs/Examples/simple-genai-gameserver.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Example Build and Run a Simple Gameserver that Connects to an Inference Server"
+linkTitle: "Build and Run a Simple GenAI Gameserver"
+date:
+publishDate:
+description: >
+  This example shows how to set up a simple game server that integrates with your inference server's GenAI endpoints. You can either interact with one GenAI endpoint via TCP, or set up two endpoints to "chat" to each other.
+---
+
+## Objectives
+
+- Set up a GenAI inference server
+- Run a simple gameserver that connects to the inference server
+- Play!
+
+## Setting up the GenAI Inference Server
+
+You will need a separate GenAI inference server. This example uses the
+[Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart) as its inference server.
+This particular inference server has the request structure for the /chat endpoint:
+
+```
+type GenAIRequest struct {
+	Context         string  `json:"context,omitempty"`
+	Prompt          string  `json:"prompt"`
+}
+```
+
+For details on how to modify the request structure for an endpoint that might use a different structure check out the [README](https://github.com/googleforgames/agones/tree/main/examples/simple-genai-server#readme).
+
+## Setting up Agones
+
+### Set up Agones on a Different Cluster than the GenAI Inference Server
```

Comment:
Create cluster + install agones shouldn't really need linking since we are inside of the agones website (and this seems obvious since we say "separate cluster"). 

What do you need to change about the configuration to run it on a different cluster? That seems to be the main difference.